decidim-forms/app/controllers/decidim/forms/concerns/has_questionnaire.rb

```diff
@@ -141,6 +143,16 @@ def tokenize(id, length: 10)
             tokenizer = Decidim::Tokenizer.new(salt: questionnaire.salt || questionnaire.id, length:)
             tokenizer.int_digest(id).to_s
           end
+
+          # This method send confirmation email with answers to user when a questionnaire is answered.
+          def send_confirmation_email
```

Comment:
Can we add a test checking that this method is invoked after answering a questionnaire and that it only sends the confirmation if the component is surveys?